### PR TITLE
T/29

### PR DESCRIPTION
**Pull request by Ardian123**

### CODE REVIEW

File: feature-rich/package.json
Comment (gorzelinski) [inline feature-rich/package.json:1]: The Feature-rich editor gives this warning about unknown integration: `image-insert-unknown-integration.` Just one option probably needs to be adjusted:

```js
```

File: ai-assistant/README.md
Comment (gorzelinski) [inline ai-assistant/README.md:1]: The AI assistant demo seems to be missing a `.gitignore` file.

File: source-code-editing/content.css
Comment (gorzelinski) [inline source-code-editing/content.css:1]: This demo gives a similar warning as the above: `image-insert-unknown-integration`.

File: user-interface-inline/content.css
Comment (gorzelinski) [inline user-interface-inline/content.css:1]: Interaction with one of the bottom two editors results in an uncaught runtime error: `ResizeObserver loop completed with undelivered notifications`. I don't know If that's expected.

File: wproofreader/README.md
Comment (gorzelinski) [inline wproofreader/README.md:1]: This demo seems to be missing a `.gitignore` file.

File: ai-assistant/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# AI Assistant editor
+
+Add powerful AI content creation tools into CKEditor. Use pre-made prompts or custom queries to have AI write, rephrase, translate or summarize your text.
```
Comment (gorzelinski) [inline ai-assistant/README.md:3]: There should probably be a comma before "or":
> Add powerful AI content creation tools into CKEditor. Use pre-made prompts or custom queries to have AI write, rephrase, translate, or summarize your text.

File: ai-assistant/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+This demo showcases most of the CKEditor features along with some premium addons: Export to PDF/Word, Import from Word.
```
Comment (gorzelinski) [inline ai-assistant/README.md:5]: It's a nitpick, but maybe we could rephrase this sentence a bit:
> This demo showcases most CKEditor features alongside some premium add-ons: Export to PDF/Word and Import from Word.

**Review by gorzelinski — COMMENTED** [review]
I looked through all the demos. Overall, they work and look well. But I found some minor problems and one error - more in the comments.

File: feature-rich/package.json
Comment (Ardian123) [reply feature-rich/package.json:1]: Changed.

File: ai-assistant/README.md
Comment (Ardian123) [reply ai-assistant/README.md:1]: Done, added.

File: internationalization/content.css
[the remark this answers is not in the document]
Comment (Ardian123) [reply internationalization/content.css:1]: Also changed.

File: mathtype/README.md
[the remark this answers is not in the document]
Comment (Ardian123) [reply mathtype/README.md:1]: Added.

File: paste-from-office-enhanced/README.md
[the remark this answers is not in the document]
Comment (Ardian123) [reply paste-from-office-enhanced/README.md:1]: Also added.

File: paste-from-office-enhanced/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Paste-from Office Enhanced editor
+
+Paste from Office Enhanced retains a wider range of formatting options compared to default paste available in the editor.
```
[the remark this answers is not in the document]
Comment (Ardian123) [reply paste-from-office-enhanced/README.md]: Added.

File: source-code-editing/content.css
Comment (Ardian123) [reply source-code-editing/content.css:1]: Insert integration changed.

File: wproofreader/README.md
Comment (Ardian123) [reply wproofreader/README.md:1]: `.gitignore` added.

File: ai-assistant/README.md (hunk 1 above)
Comment (Ardian123) [reply ai-assistant/README.md]: Comma added.

File: ai-assistant/README.md (hunk 2 above)
Comment (Ardian123) [reply ai-assistant/README.md]: Changed.

File: user-interface-inline/content.css
Comment (Ardian123) [reply user-interface-inline/content.css:1]: Found a solution. Not great, not terrible but at least it isn't displaying this error.

**Pull request merged**